pkg/coscheduling/coscheduling.go

```diff
@@ -203,27 +252,30 @@ func (cs *Coscheduling) Unreserve(ctx context.Context, state *framework.CycleSta
 	})
 }
 
-// GetPodGroupLabels will check the pod if belongs to some podGroup. If so, it will return the
-// podGroupName、minAvailable of podGroup. If not, it will return "" as podGroupName.
+// GetPodGroupLabels will check the pod if belongs to a  PodGroup. If so, it will return the
+// podGroupName、minAvailable of the PodGroup. If not, it will return the pod name and 0.
 func GetPodGroupLabels(p *v1.Pod) (string, int, error) {
 	podGroupName, exist := p.Labels[PodGroupName]
-	if !exist || podGroupName == "" {
+	if !exist || len(podGroupName) == 0 {
 		return "", 0, nil
 	}
 	minAvailable, exist := p.Labels[PodGroupMinAvailable]
-	if !exist || minAvailable == "" {
+	if !exist || len(minAvailable) == 0 {
 		return "", 0, nil
 	}
 	minNum, err := strconv.Atoi(minAvailable)
 	if err != nil {
-		klog.Errorf("GetPodGroupLabels err in coschduling %v/%v : %v", p.Namespace, p.Name, err.Error())
+		klog.Errorf("GetPodGroupLabels err in coscheduling %v/%v : %v", p.Namespace, p.Name, err.Error())
 		return "", 0, err
 	}
+	if minNum < 1 {
+		minNum = 1
+	}
```

Comment:
I am a little confused here. Why we reset the minNum = 1 if it < 1？
I suggest keeping it and add some log in `Prefilter` if we check it.